### ISSUE TITLE
Sound glitch difficult to replicate, intermittent but persistent over the last few months

### ISSUE THREAD

**Issue by Jay1** [brave/brave-browser#4787]
I'm having a sound glitch, with the sound driver Nahimic.

The bug behaviour is that from time to time when I pause the video on youtube, then I unpause the video a few moments later, the Nahimic sounds modifications are no longer working.

Nahimic is a sound mixer panel let me show you ; 

![nahimic](https://user-images.githubusercontent.com/1072434/59227308-0ddc9200-8ba3-11e9-82c6-4200ea905a45.PNG)

The bug is really difficult to replicate because it's not every time I pause/unpause. Additionally, when the bug happens, even if I pause/unpause real fast, it doesn't fix the bug either, it takes a while. I don't quite understand what time has anything to do with this, it's a complete mystery for me right now. I tried to spam the pause/unpause while looking at the console, it's very unclear what is glitching behind the scene. 

1. So the bug is always triggered by pausing the video
2, It's always fixed by pausing/unpausing BUT only after a while not instantly

I don't know so far if it affects a lot of audio mixer panels out there, but it may be worth looking at it.

- The bug is related to **every version of Brave** so far, (all the branches I tried them all)
- I'm on Windows 10

I opened other web browsers, and the sound is fine on youtube (even during the bug is happening on Brave. Therefore the audio could be perfect on Opera and bugging on Brave at the same time). 

It's just the interaction in between Nahimic and Brave that seems to be conflicting with one another. I have yet to discover why, I'm sorry I would like to provide more valuable information to help you on your quest, but I'm not very experienced with browser development.

**Comment by rebron** [comment]
@Jay1 Did you try with shields down if that's effecting your experience?

**Comment by Jay1** [comment]
> @Jay1 Did you try with shields down if that's effecting your experience?

First of all, sorry for the delay!

After further testing and more investigation, the bug is still present, I'm using the nightly build atm to get the fastest patching rate. Sometimes you touch something in the code and it fixes something else so it's really important for me to up to date.

I tried to fix the issue while playing with the shields, on/off, refresh the browser with on/off. So far it's not conclusive, I don't see any difference. The bug is not affected by the shields whatsoever.

**Comment by kjozwiak** [comment]
> I opened other web browsers, and the sound is fine on youtube (even during the bug is happening on Brave. Therefore the audio could be perfect on Opera and bugging on Brave at the same time).

I'm assuming you've also tried it on Chrome as well correct? Just double checking to make sure it's related to Brave and not a general issue with Chromium.

**Comment by Jay1** [comment]
> > I opened other web browsers, and the sound is fine on youtube (even during the bug is happening on Brave. Therefore the audio could be perfect on Opera and bugging on Brave at the same time).
> 
> I'm assuming you've also tried it on Chrome as well correct? Just double checking to make sure it's related to Brave and not a general issue with Chromium.

Absolutely, chrome works perfectly fine while it's bugging on Brave at the exact same time. I made sure to confirm that

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
